packages/cisco_aironet/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,302 @@
+      patterns:
+        - "Signature information; AP %{MAC:destination.mac}, alarm ON, %{SIGNATURE:threat.indicator.description}, track %{DATA}precedence %{INT}, hits %{INT}, slot %{INT}, channel %{INT}, most offending MAC %{MAC:source.mac}"
+      pattern_definitions:
+        SIGNATURE: "%{DATA} sig %{DATA}"
+      ignore_failure: true
+  - set:
+      description: LWAPP-4-SIG_INFO1
+      field: threat.indicator.type
+      if: ctx._temp_?.reason == 'LWAPP-4-SIG_INFO1' && ctx.threat?.indicator?.description != null && ctx.threat.indicator.description != ""
+      value: process
+      ignore_failure: true
+  ###
+  - grok:
+      description: DOT1X-4-MAX_EAPOL_KEY_RETRANS
+      field: message
+      if: ctx._temp_?.reason == 'DOT1X-4-MAX_EAPOL_KEY_RETRANS'
+      patterns:
+        - "client %{MAC:client.mac}"
+      ignore_failure: false
+  ###
+  - grok:
+      description: RRM-3-RRM_LOGMSG
+      field: message
+      if: ctx._temp_?.reason == 'RRM-3-RRM_LOGMSG'
+      patterns:
+        - "Client not found: %{MAC:client.mac}"
+        - "AP:  %{MAC:host.mac}"
+      ignore_failure: false
+  ###
+  - grok:
+      description: DOT1X-3-ABORT_AUTH
+      field: message
+      if: ctx._temp_?.reason == 'DOT1X-3-ABORT_AUTH'
+      patterns:
+        - "client %{MAC:client.mac} Abort Reason:%{DATA:event.reason}$"
+      ignore_failure: false
+  ###
+  - grok:
+      description: LOG-3-Q_IND, INVALID_WPA_KEY_STATE
+      field: message
+      if: ctx._temp_?.reason == 'LOG-3-Q_IND' ||
+        ctx._temp_?.reason == 'DOT1X-3-INVALID_WPA_KEY_STATE'
+      patterns:
+        - "version %{INT:cisco.eapol.version:int}, type %{INT:cisco.eapol.type:int}, descriptor %{INT:cisco.eapol.descriptor:int}, client %{MAC:client.mac}"
+        - "client %{MAC:client.mac}"
+      ignore_failure: false
+  ###
+  - grok:
+      description: APF-6-USER_NAME_CREATED, APF-6-USER_NAME_DELETED
+      field: message
+      if: ctx._temp_?.reason == 'APF-6-USER_NAME_CREATED' ||
+        ctx._temp_?.reason == 'APF-6-USER_NAME_DELETED'
+      patterns:
+        - "Username entry \\(%{DATA:user.name}\\)%{DATA}mobile %{MAC:client.mac}"
+      ignore_failure: false
+  ###
+  - grok:
+      description: DTL-4-ARP_ORPHANPKT_DETECTED, Dot1x_NW_MsgTask_4
+      field: message
+      if: ctx._temp_?.reason == 'DTL-4-ARP_ORPHANPKT_DETECTED' ||
+        ctx._temp_?.reason == 'LOG-4-Q_IND'
+      patterns:
+        - "STA\\(Target MAC Address\\) \\[%{MAC:client.mac}.*?\\] %{DATA:event.reason}\\(Source IP Address\\) %{IP:client.ip}%{DATA}\\(Destination IP Address\\) %{IP:server.ip}"
+      ignore_failure: false
+  ###
+  # Client MAC
+  - grok:
+      field: message
+      ignore_missing: true
+      ignore_failure: true
+      patterns:
+        - "client %{MAC:client.mac}"
+  # Mac address cleanup
+  - uppercase:
+      field: client.mac
+      ignore_missing: true
+  - gsub:
+      field: client.mac
+      pattern: '[-:.]'
+      replacement: '-'
```

```suggestion
      pattern: '[:.]'
      replacement: '-'
```
Same for the others that are `s/[-:.]/-/g`.